web/README.md

```diff
@@ -1,32 +1,96 @@
-# voron-web, aka moron :%)
+# voron Web Front
 
-Built with:
+Our Web front-end is built with:
 
--   `yarn`
+-   `yarn` | `npm` (please choose one to use, do not mix)
```

Comment:
since `yarn` is not supported in `.tool-versions`, it's probably better to recommend it